src/base-input.js

```diff
@@ -131,14 +131,25 @@ export default class BaseInput extends LitElement {
    * @returns {string} Replaces innerHTML with empty string.
    */
   handleClickClear() {
+    let click = null;
+
     this.inputElement.value = "";
     this.value = "";
     this.classList.remove("passwordIcon--show");
     this.focus();
 
+    click = new CustomEvent('clearElement', {
+      bubbles: true,
+      composed: true,
+    });
+
     if (!this.noValidate) {
       this.validate();
     }
+
+    this.dispatchEvent(click);
+
+    console.log(click)
```

Comment:
Remove console.log

Reply:
The console log is there to illustrate the return of the event. This will be removed once this is ready to merge.